crates/crypto/src/lib.rs

```diff
@@ -1,11 +1,12 @@
-use calimero_primitives::identity::PublicKey;
-use ed25519_dalek::SigningKey;
+use calimero_primitives::identity::{PrivateKey, PublicKey};
+use ed25519_dalek::hazmat::ExpandedSecretKey;
```

Comment:
I'd [avoid using this directly](https://docs.rs/ed25519-dalek/latest/ed25519_dalek/hazmat/index.html#-warning-hazmat)